modules/diskq/tests/test_diskq_truncate.c

```diff
@@ -257,6 +258,123 @@ Test(diskq_truncate, test_diskq_truncate_on_push)
   _save_diskqueue(q);
 }
 
+static void
+_assert_cursors_are_at_start(LogQueue *q)
+{
+  QDisk *qdisk = ((LogQueueDisk *)q)->qdisk;
+
+  cr_assert_eq(qdisk->hdr->read_head, QDISK_RESERVED_SPACE, "Read head was not reset!");
+  cr_assert_eq(qdisk->hdr->write_head, QDISK_RESERVED_SPACE, "Write head was not reset!");
+  cr_assert_eq(qdisk->hdr->backlog_head, QDISK_RESERVED_SPACE, "Backlog head was not reset!");
+}
+
+Test(diskq_truncate, test_diskq_truncate_size_ratio_default)
+{
+  const gint truncate_size_ratio = (gint)(1100000 * 0.01);
```

Comment:
minor, it's a bit confusing that this is the same name as the option itself: truncate_size_ratio.
The units of the option is percentage, while this constant is in bytes, like disk-buf-size.

please rename it.

Reply:
Thanks, fixed.